oracle-plugin/src/e2e-test/java/io.cdap.plugin/oracle/stepsdesign/Oracle.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.oracle.stepsdesign;
+
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.OracleClient;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.sql.SQLException;
+
+/**
+ *  Oracle Plugin related step design.
+ */
+public class Oracle implements CdfHelper {
+
+  @Then("Click on preview data for Oracle sink")
+  public void clickOnPreviewDataForOracleSink() {
+    openSinkPluginPreviewData("Oracle");
+  }
+
+
+  @Then("Get count of no of records transferred to target Oracle Table")
```

Comment:
Ideally this should named as "Validate number of records in target table matches with the number of records written by the pipeline."

Reply:
We have a function below this for validating if the number of records match. This function is just to get the count of records now available at the sink side.